src/app/app-routing.module.ts

```diff
@@ -12,7 +12,10 @@ const appRoutes: Routes = [
 
 @NgModule({
   imports: [
-    RouterModule.forRoot(appRoutes)
```

Comment:
There are a few small changes to simplify deployment to github pages (e.g., https://ncsa.github.io/phylodiver-viz/#/demo). I might come back and clean them up in a later PR.